atd-vze/src/Components/GridTable.js

```diff
@@ -148,14 +266,16 @@ const GridTable = ({ title, query }) => {
     data[query.table].map((row, index) =>
       dataEntries.push(
         <tr key={index}>
-          {query.columns.map(column => (
-            <td>
+          {query.columns.map((column, ci) => (
+            <td key={ci}>
```

Comment:
also note, if there are any other key's defined with index iterators on the page, the key will no longer be unique and you'll get a console warning. I usually do something like `key={`GridTableColumn-${ci}`}` or soomething that describes the component so its more unique